torchao/prototype/low_bit_optim/cpu_offload.py

```diff
@@ -13,6 +13,7 @@ def __init__(
         optimizer_class: Type[Optimizer] = torch.optim.AdamW,
         *,
         offload_gradients: bool = False,
+        device: str = "cuda",
```

Comment:
Similarly to the above comment. Can we auto-detect device instead?

Reply:
Add auto-detect available devices in torchao.utils. THX for ur review comment.